contracts/src/access/ownable_two_step.rs

```diff
@@ -196,6 +188,22 @@ impl IOwnable2Step for Ownable2Step {
 }
 
 impl Ownable2Step {
+    /// Checks if the [`msg::sender`] is set as the owner.
+    ///
+    /// Re-export of [`Ownable::only_owner`].
+    ///
+    /// # Arguments
+    ///
+    /// * `&self` - Read access to the contract's state.
+    ///
+    /// # Errors
+    ///
+    /// If called by any account other than the owner, then the error
+    /// [`OwnableError::UnauthorizedAccount`] is returned.
+    pub fn only_owner(&self) -> Result<(), OwnableError> {
```

Comment:
Why are you re-exporting only owner back? It was an explicit [design decision](https://github.com/OpenZeppelin/rust-contracts-stylus/pull/352#discussion_r1832280260). I think we should at least discuss it.
And if we do decide to add it, we should change all usages of `only_owner` to use this?

Reply:
After a second thought, I strongly believe that the interface of `Ownable2Step` should be same as in Solidity, so `only_owner()` should be possible to use on `Ownable2Step` contract without the need of accessing it via contract's attributes (better DevEx). But I am open to discussion on this.

Reply:
Wel.... this isn't part of the public interface, it is just internal. I do agree that it is better DX, but my concern is that this will result in a bunch of code repetition if we follow this pattern(and we should be consistent)

Reply:
I generally lean towards better DevEx (in this case reexporting `only_owner`), but not at the price of having to update (almost) all existing contracts that do not follow the proposed convention.

Maybe if we start getting complaints from devs we can rethink?

Reply:
Reverted